main.c

```diff
@@ -771,7 +776,7 @@ void setRank() {
   }
 
   /* Searches for values in the top 5 ranks that are lower than the
-   the new player score. */
+   the new player score. 
```

Comment:
This breaks the commit and makes it hard to re-enable the code later.
To disable dead code (if absolutely necessary), use:
```c
#if 0
Dead code goes here
```

Reserve `//` and `/* */` for actual comments.